docs/Citus README.md

```diff
@@ -0,0 +1,42 @@
+# SQLancer for Citus (PostgreSQL extension)
+
+SQLancer (Synthesized Query Lancer) is a tool to automatically test Database Management Systems (DBMS) in order to find logic bugs in their implementation. More information about the tool can be found in the [SQLancer README](https://github.com/sqlancer/sqlancer).
+
+The Citus implementation of SQLancer supports the Ternary Logic Query Partitioning (TLP) test oracle.
+
+# Setting up
+
+Instructions for setting up SQLancer can be found [here](https://github.com/sqlancer/sqlancer#getting-started).
+
+Requirements for Citus:
+* PostgreSQL & Citus - The steps required to build Citus from source can be found [here](https://github.com/citusdata/citus/blob/master/CONTRIBUTING.md).
+Optional Requirements for Citus:
+* [pgenv](https://github.com/thanodnl/pgenv) (for easier management of PostgreSQL versions)
+* [citus_dev](https://github.com/citusdata/tools/tree/develop/citus_dev) (for easier configuration of Citus environment)
+
+# Using SQLancer
+
+The following commands run the Citus implementation of SQLancer using Ternary Logic Query Partitioning (TLP):
+
+```
+cd target
+java -jar SQLancer-0.0.1-SNAPSHOT.jar --num-threads 4 citus --oracle QUERY_PARTITIONING
+```
+
+How to configure the run and how to interpret the output is explained [here](https://github.com/sqlancer/sqlancer#using-sqlancer).
+
+# Contributing
+
+The instructions for setting up a development environment for contributing to SQLancer can be found [here](https://github.com/sqlancer/sqlancer/blob/master/CONTRIBUTING.md).
+
+## Updating expected Citus errors list
+
+The `CitusCommon.java` file in the `src/sqlancer/citus/gen/` directory should be continuously updated to reflect the currently unsupported functionalities and the willingness to ignore active bugs. The `addCitusErrors()` method adds errors that should be ignored if raised, either because a SQL command generated by SQLancer is not supported by Citus, or because a bug that has already been identified is redundantly re-appearing. 
```

Comment:
This could use a small description on explaining the concept of expected errors, why it is important and then how to update. Now this might be hard to understand its significance if someone on the citus team without background knowledge needs to update the expected errors.